openedx/core/djangoapps/api_admin/models.py

```diff
@@ -35,6 +42,8 @@ class ApiAccessRequest(TimeStampedModel):
     reason = models.TextField(help_text=_('The reason this user wants to access the API.'))
     company_name = models.CharField(max_length=255, default='')
     company_address = models.CharField(max_length=255, default='')
+    base_url = models.CharField(max_length=255, default='')
```

Comment:
Oh, sorry, I'd misread the docs and thought that `get_absolute_url` was defined in the base class, rather than being a hook for Django to call. :+1: